pypdf/generic/_base.py

```diff
@@ -337,17 +337,24 @@ def readFromStream(
         return IndirectObject.read_from_stream(stream, pdf)
 
 
-class FloatObject(decimal.Decimal, PdfObject):
+class FloatObject(float, PdfObject):
     def __new__(
-        cls, value: Union[str, Any] = "0", context: Optional[Any] = None
+        cls, value: Union[str, Any] = "0.0", context: Optional[Any] = None
```

Comment:
Could we make the default `0.0` and have another `isinstance(value, float)` later? Maybe at first?

Reply:
not sure isinstance(value, float) required